src/main/java/org/zalando/pazuzu/api/ApiUrls.java

```diff
@@ -0,0 +1,18 @@
+package org.zalando.pazuzu.api;
+
+public final class ApiUrls {
+
+    private ApiUrls() {}
+
+    /*
+     * Ant matcher used for auth config.
+     */
+    public static final String BASE_API_ANT_MATCHER = "/api/**";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2752585987/issues/source?bid=3131965&fileBranchId=3348590#l10)